packages/rocketchat-ui-account/account/accountProfile.coffee

```diff
@@ -106,3 +106,9 @@ Template.accountProfile.onRendered ->
 Template.accountProfile.events
 	'click .submit button': (e, t) ->
 		t.save()
+	'click .logoutOthers button': (event, templateInstance) ->
+		Meteor.logoutOtherClients (error) -> 
+			if error
+				toastr.error error.reason
+			else
+				toastr.success t('Logged out of other clients successfully')
```

Comment:
This string must also be added to `i18n/en.18n.json`. Please change it to `t('Logged_out_of_other_clients_successfully')` and add it to the translations file.